doc/options.jax

```diff
@@ -3484,7 +3515,8 @@ Note 1番目の形式では、行全体がオプション指定に使われる
 
 	例: >
 		:set formatexpr=mylang#Format()
-<	これはautoload/mylang.vim中の関数mylang#Format()を呼ぶ。|autoload|
+<	これは 'runtimepath' 内にある autoload/mylang.vim 中の関数
```

Comment:
原文は下記ですが、「'runtimepath' の」があった方が良さそうでしたので追加しました。

<	This will invoke the mylang#Format() function in the
	autoload/mylang.vim file in 'runtimepath'. |autoload|